docs/job_types/job_python3.md

```diff
@@ -2,7 +2,7 @@
 "python3" job type is intended to handle the calls to your Python function.
 Racetrack will wrap it up in a web server.
 
-Set `lang: python3:latest` in your `job.yaml` manifest file in order to use this type of job.
+Set `jobtype: python3:latest` in your `job.yaml` manifest file in order to use this type of job.
```

Comment:
Let's remove job_python3.md in here. It's been moved to https://github.com/TheRacetrack/plugin-python-job-type/blob/master/docs/job_python3.md